### DISCUSSION TITLE
Access qbittorent config

### DISCUSSION THREAD

**Discussion by Kitsunejasutin** [tteck/Proxmox#2160] [Help Needed] [answered]
I'll make it short as possible but when I did a customization and applied a theme and I guess I borked the config resulting to the WEB UI stuck here.
![image](https://github.com/tteck/Proxmox/assets/73803767/808d8531-7130-425b-91c4-b633051e7c46)

Where can I directly access the config file so I can remove the custom UI I set. Thank you!

**Comment by tteck** [comment, marked as answer]
```
nano /.config/qBittorrent/qBittorrent.conf
```

**Reply by Kitsunejasutin** [reply]
ohhh, I guess all configs is setup like this. This is noted, thank you!
